src/thirdparty/fmt/core.h

```diff
@@ -1571,12 +1571,10 @@ class format_arg_store
 
   friend class basic_format_args<Context>;
 
+  static constexpr unsigned long long type_encoding = detail::encode_types<Context, Args...>();
```

Comment:
Should we put this in a patch file and/or add comments/tags in the file to make it easier to see this patch?

Reply:
Yea I'll push that in a bit.. was just so excited to finally track this down.